extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -378,6 +383,24 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
   }
 
+  /** @private */
+  maybeSetStoryNextUp_() {
+    const autoAdvanceAttr = this.element.getAttribute('auto-advance-after');
+    const storyNextUpParam = Services.viewerForDoc(this.element).getParam(
+      'storyNextUp'
+    );
+    if (
+      storyNextUpParam !== null &&
+      autoAdvanceAttr === null &&
```

Comment:
nit: switch these so they read in the same order as the const declarations above.

Reply:
Done. Good call.